website/main.js

```diff
@@ -0,0 +1,20 @@
+function renderPage(divName, menuItem) {
+    // hide any visible content divs
+    var contentDivs = document.getElementsByClassName('content');
+    for (var i = 0; i < contentDivs.length; i++) {
+        if (!contentDivs[i].classList.contains('hidden')) {
+            contentDivs[i].classList.add('hidden');
+        }
+    }
```

Comment:
I'm personally a fan of writing declarative JS, so I would use `forEach` here.
```suggestion
    Array.from(contentDivs).forEach(contentDiv => {
        if (!contentDiv.classList.contains('hidden')) {
            contentDiv.classList.add('hidden');
        }
    });
```